gravatar-ui/src/main/java/com/gravatar/ui/components/atomic/ViewProfileButton.kt

```diff
@@ -50,11 +51,28 @@ public fun ViewProfileButton(
     modifier: Modifier = Modifier,
     textStyle: TextStyle = MaterialTheme.typography.titleSmall.copy(color = MaterialTheme.colorScheme.onBackground),
     inlineContent: @Composable ((String) -> Unit)? = { DefaultInlineContent(textStyle.color) },
+) {
+    ViewProfileButton(
+        stringResource(R.string.view_profile_button),
+        profile.profileUrl().url.toString(),
+        textStyle,
+        modifier,
+        inlineContent,
+    )
+}
+
+@Composable
+private fun ViewProfileButton(
```

Comment:
I think we can rename that to something more generic, since we can change the text and the url, this button can do other actions.

That reminded me, we should create an "EditProfileButton" component as well (probably needed for Pocket Casts integration).